simphony_metadata.yml

```diff
@@ -13,41 +13,41 @@ Resources:
 CUDS_KEYS:
 
   CUDS_ITEM:
-    parent: 
+    parent:
     CUBA.UUID:
       scope: CUBA.SYSTEM
-    CUBA.DATA:  
-      
+    CUBA.DATA:
+
   CUDS_COMPONENT:
     definition: Base data type for the CUDS components
     parent: CUBA.CUDS_ITEM
     CUBA.DESCRIPTION:
```

Comment:
Yes, description is changed to definition. If CUBA.DESCRIPTION is used in cuds_component, it should be defined somewhere, probably in simphony_metadata as I assume it's going to hold more complex information than basic CUBA keys defined in cuba.yml